lib/internal/util.js

```diff
@@ -18,7 +18,15 @@ exports.printDeprecationMessage = function(msg, warned) {
 };
 
 exports.error = function(msg) {
-  console.error(`${prefix}${msg}`);
+  const fmt = `${prefix}${msg}`;
+  if (arguments.length > 1) {
+    const args = new Array(arguments.length);
+    args[0] = fmt;
+    for (let i = 1; i < arguments.length; ++i)
+      args[i] = arguments[i];
```

Comment:
Well if for no other reason than I think spread is slow in v8 still? I am not 100% sure if that changed in v8 4.7 or not...